### ISSUE TITLE
Playtest lobby crash

### ISSUE THREAD

**Issue by netnazgul** [OpenRA/OpenRA#14725]
Got a crash on 20180102 version once, when entering the multiplayer server lobby.

Got it on my stream here: https://youtu.be/a20zslCLGZI?t=7877

Exception log:
```
OpenRA
 engine version playtest-20180102
Red Alert mod version playtest-20180102
Date: 2018-01-06 21:11:50Z
Operating System: Windows (Microsoft Windows NT 6.2.9200.0)
Runtime Version: .NET CLR 4.0.30319.42000
Exception of type `System.IO.EndOfStreamException`: Чтение после конца потока невозможно.
   в System.IO.BinaryReader.ReadString()
   в OpenRA.Order.Deserialize(World world, BinaryReader r)
   в OpenRA.Network.OrderIO.ToOrderList(Byte[] bytes, World world)
   в OpenRA.Network.ReplayRecorder.IsGameStart(Byte[] data)
   в OpenRA.Network.ReplayRecorder.Receive(Int32 clientID, Byte[] data)
   в OpenRA.Network.EchoConnection.Receive(Action`2 packetFn)
   в OpenRA.Network.OrderManager.TickImmediate()
   в OpenRA.Sync.<CheckSyncUnchanged>c__AnonStorey0.<>m__0()
   в OpenRA.Sync.CheckSyncUnchanged[T](World world, Func`1 fn)
   в OpenRA.Sync.CheckSyncUnchanged(World world, Action fn)
   в OpenRA.Game.InnerLogicTick(OrderManager orderManager)
   в OpenRA.Game.LogicTick()
   в OpenRA.Game.Loop()
   в OpenRA.Game.Run()
   в OpenRA.Game.InitializeAndRun(String[] args)
   в OpenRA.WindowsLauncher.RunGame(String[] args)
```

**Comment by pchote** [comment]
Looks related to #14713.  We must still have something wrong in the order (de)serialization.

**Comment by pchote** [comment]
Closing as fixed by #14748.  If this is actually something different then I hope we'll receive new reports on the next playtest.
